Fulfillment/UserInterface/03b_PackagingTypes.md

```diff
@@ -0,0 +1,357 @@
+
+- *Max value*  
+  If desired, the maximal monetary value allowed for the packaging type can be configured.
+    - [ADD PRICE]  
+    Click this button to enter the base price and scale prices. The *Edit price* window is displayed, see [Edit price](#edit-price).
+    - *Base price*  
+    Base price defined for the product. This field is read-only and can only be edited in the *Edit price* window.
+    - *Current price range*  
+    Price range defined for the product. This field is read-only and can only be edited in the *Edit price* window.
+    - *# Scale prices*  
+    Number of scale prices defined for the product. This field is read-only and can only be edited in the *Edit price* window.
+    - ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit)  
+    Click this button to edit a configured price. This button is displayed when hovering the mouse over a configured price. The *Edit price* window is displayed, see [Edit price](#edit-price).
+    - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete)  
+    Click this button to deleted the configured price. This button is displayed when hovering the mouse over a configured price.
+
+[comment]: <> (Stimmt das so? Button SAVE funktioniert nicht. Bug gemeldet.)
+
+- *Carrier*  
+    Enter the preferred carrier for the packaging type. Numbers, letters or a combination of characters can be used.
+
+- ![Toggle](../../Assets/Icons/Toggle.png "[Toggle]") *Express*  
+    Enable this toggle to set the shipping type for the packaging type to express. Disable the toggle to set the shipping type for the packaging type to standard. By default, this toggle is disabled.  
+
+- *Packaging type identifier*  
+    Enter the packaging type identifier. Numbers, letters or a combination of characters can be used.
+
+- *Ship-to country*  
+    Click the drop-down list and select the country where the packaging type is shipped to.
+
+- *Priority*  
+    Enter a number to set the priority for the packaging type. The priority value is used by the system to determine the packaging type to be used for a shipment when the criteria of two or more packaging types match. The lower the number, the higher the priority.
+
+- *Additional services (shipping method)*  
+    Enter the additional services offered. Numbers, letters or a combination of characters can be used.
+
+
+## Edit packaging type
+
+*Fulfillment > Settings > Tab PACKAGING TYPES > Select packaging type*
+
+![Edit packaging type](../../Assets/Screenshots/Fulfillment/Settings/PackagingTypes/EditPackagingType.png "[Edit packaging type]")
+
+- ![Back](../../Assets/Icons/Back02.png "[Back]") (Back)   
+    Click this button to close the *Edit packaging type* view and return to the list of packaging types. All changes are rejected.
+
+- *Channel*  
+    Click the drop-down list to select the channel. All available  are displayed in the drop-down list. By default, the *Actindo Basic* channel is preselected.
```

Comment:
```suggestion
    Click the drop-down list to select the channel. All available channels are displayed in the drop-down list.
```
Würde den Actindo Basic channel hier raus lassen, wenn das in fulfillment eh nicht mehr vorkommen soll...